Sources/Testing/Traits/ConditionTrait.swift

```diff
@@ -149,6 +149,32 @@ extension Trait where Self == ConditionTrait {
     Self(kind: .conditional(condition), comments: Array(comment), sourceLocation: sourceLocation)
   }
 
+  /// Constructs a condition trait that disables a test if an environment
+  /// variable is not present.
+  ///
+  /// - Parameters:
+  ///   - comment: An optional comment that describes this trait.
+  ///   - sourceLocation: The source location of the trait.
+  ///   - environmentName: The name of an environment variable. If it is found
+  ///     the current environment, the trait allows the test to run.
+  ///     Otherwise, the testing library skips the test. The value of
+  ///     the environment variable is not inspected.
+  ///
+  /// - Returns: An instance of ``ConditionTrait`` that checks for the provided
+  /// environment variable.
+  @_spi(Experimental)
+  public static func enabled(
+    ifEnvironmentPresent environmentName: String,
```

Comment:
A single environment variable is not "an environment" so we need to bikeshed this parameter label and name.